src/app/enums/enums.ts

```diff
@@ -12,3 +12,11 @@ export const languageProficiencyLevels: LanguageProficiency[] = [
   {name: 'Professional proficiency', proficiency: 4},
   {name: 'Native speaker', proficiency: 5}
 ];
+
+export const skillProficiencyLevels: LanguageProficiency[] = [
+  {name: 'Beginner', proficiency: 1},
+  {name: 'Basic knowledge', proficiency: 2},
+  {name: 'Intermediate knowledge', proficiency: 3},
+  {name: 'Professional', proficiency: 4},
+  {name: 'Expert', proficiency: 5}
```

Comment:
Hmm, tried to implement this but since the automcomplete-dropdown does not support translations this is currently not possible. We need a separate task on asana for this.